chart/mixnode/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
The directory should be called `charts` instead of `chart` (there might be more than one chart) for the publish and deploy scripts to work.